doc.go

```diff
@@ -186,6 +189,19 @@ func suggestPackages(tok string) (cnt int) {
 
 // Expand GOROOT and GOPATH with respect to some dirPath.
 func makePaths(dirPath string) (paths []string) {
+	if strings.HasPrefix(dirPath, ".") {
```

Comment:
Just add a comment here that this is for resolving local paths. Otherwise, I'm left thinking if this is to handle hidden directories or something.